R/regularize_vpts.R

```diff
@@ -1,3 +1,5 @@
+globalVariables(c("x","y","closestt"))
```

Comment:
can this be removed?

Reply:
As far as I can see it it not easy, it might be possible to do something with `rlang::sym` but the code would become very unreadable. It needed for the line where `closest` is used (I see i made a typo here). For other dplyr stuff generally `.data` works but here its not really possible as far as I can see and they are also not exported from `dplyr`

Reply:
The note here is a result of this typo:
https://github.com/adokter/bioRad/actions/runs/5003819799/jobs/8965543985#step:9:96